.github/workflows/checks.yml

```diff
@@ -2,7 +2,7 @@ name: Checks
 
 on:
   pull_request:
-    branches: [main]
+    branches: "*"
```

Comment:
you can actually just remove the branches key entirely and it will have the same effect